StripeIdentity/StripeIdentity/Source/NativeComponents/ViewControllers/PhoneOtpViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  PhoneOtpViewController.swift
+//  StripeIdentity
+//
+//  Created by Chen Cen on 6/14/23.
+//
+
+@_spi(STP) import StripeUICore
+import UIKit
+
+class PhoneOtpViewController: IdentityFlowViewController {
+    private let phoneOtpContent: StripeAPI.VerificationPageStaticContentPhoneOtpPage
+
+    let phoneOtpView: PhoneOtpView
+
+    var flowViewModel: IdentityFlowView.ViewModel {
+        return .init(
+            headerViewModel: .init(
+                backgroundColor: .systemBackground,
+                headerType: .plain,
+                titleText: phoneOtpContent.title
+            ),
+            contentView: phoneOtpView,
+            buttons: [
+                .init(
+                    text: phoneOtpContent.resendButtonText,
+                    state: {
+                        switch phoneOtpView.viewModel {
+                        case .InputtingOTP:
+                            return .enabled
+                        case .SubmittingOTP:
+                            return .disabled
+                        case .ErrorOTP:
+                            return .enabled
+                        case .RequestingOTP:
+                            return .loading
+                        case .RequestingCannotVerify:
+                            return .disabled
+                        }
+                    }(),
+                    isPrimary: false,
+                    didTap: {
+                        self.generateOtp()
```

Comment:
I'd double-check if this is not a retain cycle

So `PhoneOtpViewController retains `IdentityFlowView.ViewModel` and `IdentityFlowView.ViewModel` retains `PhoneOtpViewController`

Reply:
updated to `weak self`